tests/pytests/common.py

```diff
@@ -320,14 +320,17 @@ def get_vecsim_debug_dict(env, index_name, vector_field):
     return to_dict(env.cmd(debug_cmd(), "VECSIM_INFO", index_name, vector_field))
 
 
-def forceInvokeGC(env, idx = 'idx', timeout = None):
+def forceInvokeGC(env, idx='idx', timeout=None, run_in_background=False):
     waitForRdbSaveToFinish(env)
-    if timeout is not None:
-        if timeout == 0:
-            env.debugPrint("forceInvokeGC: note timeout is infinite, consider using a big timeout instead.", force=True)
+    if run_in_background:
+        env.cmd(debug_cmd(), 'GC_FORCEBGINVOKE', idx)
+    elif timeout is not None:
+        # Note: timeout==0 means infinite (no timeout)
         env.cmd(debug_cmd(), 'GC_FORCEINVOKE', idx, timeout)
     else:
         env.cmd(debug_cmd(), 'GC_FORCEINVOKE', idx)
```

Comment:
so essintialy we can't pass both `timeout` and `run_in_background=False`
I suggest splitting this into 2 functions for each use case.